src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -294,7 +294,7 @@
           short-summary: Maximum nodes count used for autoscaler, when "--enable-cluster-autoscaler" specified. Please specify the value in the range of [1, 1000].
         - name: --vm-set-type
           type: string
-          short-summary: Agent pool vm set type. VirtualMachineScaleSets, AvailabilitySet or VirtualMachines(internal use only).
+          short-summary: Agent pool vm set type. VirtualMachineScaleSets, AvailabilitySet or VirtualMachines(Preview).
```

Comment:
How about adding the `is_preview` flag for this parameter instead of `(Preview)` here?

Reply:
What do you mean?

Reply:
shouldn't be a blocker

Reply:
@zqingqing1 What I mean is https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#preview-commands-and-arguments

Reply:
The --vm-set-type is not preview parameter, only one of the value is Preview.

Reply:
Got it, thanks~